### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by johnmcollier**
Signed-off-by: John Collier <jcollier@redhat.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+You can easily add this footer to your commits by adding `-s` when running `git commit`.When you think the code is ready for review, create a pull request and link the issue associated with it.
+
+Owners of the repository will watch out for and review new PRs. 
+
+By default for each change in the PR, GitHub Actions and OpenShift CI will run checks against your changes (linting, unit testing, and integration tests)
+
+If comments have been given in a review, they have to be addressed before merging.
+
+After addressing review comments, don’t forget to add a comment in the PR afterward, so everyone gets notified by Github and know to re-review.
```
Comment (mike-hoang) [inline CONTRIBUTING.md:79]:
```suggestion
After addressing review comments, don’t forget to add a comment in the PR afterward, so everyone gets notified by Github and knows to re-review.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+Thank you for your interest in contributing to the Devfile Registry Operator! We welcome your additions to this project.
+
+## Code of Conduct
+
+Before contributing to this repository for the first time, please review our project's [Code of Conduct](https://github.com/devfile/api/blob/main/CODE_OF_CONDUCT.md)
+
+## How to Contribute:
```
Comment (mike-hoang) [inline CONTRIBUTING.md:9]: I think we can remove the colon here
```suggestion
## How to Contribute
```

**Review by mike-hoang — APPROVED** [review]
Super minor nits -- I think the actual content looks good

**Comment by johnmcollier** [comment]
@mike-hoang Review comments addressed.

**Pull request merged**